src/util/sha2.rs

```diff
@@ -0,0 +1,69 @@
+//! fuzztarget-only Sha2 context with a dummy Sha256 and Sha512 hashers.
+
+#[cfg(feature = "fuzztarget")]
```

Comment:
We can drop this (and the `cfg` line further down) since the whole module is feature gated.

Reply:
Fixed.